.travis.yml

```diff
@@ -25,3 +25,4 @@ env:
     - TEST_SET=reconfig
     - TEST_SET=solo SKIP_STACK=true
     - TEST_SET=stack
+    - TEST_SET=unit SKIP_INSTALL=true SKIP_STACK=true
```

Comment:
I think we should consider combining this with topology (and possibly renaming to 'base'). The problem is that each high-level entry is fairly expensive in terms of startup-costs, and so adding lots of high-level ones isn't always the the right level of granularity. 

test_base could just call test_topology and test_unit  -- so the underlying modularity is preserved.

Reply:
Created test_base amd calling test_topology and test_unit there.